client/app/src/main/java/com/microsoft/research/karya/data/manager/RetrofitFactory.kt

```diff
@@ -8,7 +8,7 @@ import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitFactory {
-    private val SERVER_URL = "http://192.168.0.105:4040"
+    private val SERVER_URL = "https://karyaboxtest.westcentralus.cloudapp.azure.com"
```

Comment:
@anuragshukla06 do we want to expose the server url to everyone?

Reply:
Its a test server and one cannot use the APIs without the access code. I kept it in in dev branch so that we can easily test the app but sure, I will remove it

Reply:
No, I'm okay with this being here. I was just worried that if MSR won't have any issues around it. We can definitely keep this here.

Reply:
No it is fine